第1章終了

shmn7iii opened the pull request:
# 第1章終了
成果URL：https://gentle-shore-22479.herokuapp.com/

## 備考
- Cloud9ではなくローカル環境で作業
- developブランチで作業した関係でGit,GitHub関連の部分は適宜読み替え/読み飛ばし

shmn7iii committed:
create new rails app

shmn7iii committed:
change directory structure

shmn7iii committed:
bundle lock --add-platform x86_64-linux

shmn7iii committed:
update readme

shmn7iii committed:
add heroku-url to readme

p0pemaru commented on Gemfile:
```diff
@@ -0,0 +1,32 @@
+
+group :production do
+  gem 'pg', '1.1.4'
+end
```
末尾に空行を入れることで`-`マークが出なくなります。
ファイル保存時に自動的に空行を必要に応じて挿入するエディタの機能などを活用するのがおすすめです！
cf. https://qiita.com/norikt/items/83674fadd79a88bf7824

shmn7iii replied:
なるほど、この機能は知りませんでした。ありがとうございます！

p0pemaru commented on config/credentials.yml.enc:
```diff
@@ -0,0 +1 @@
+uVbjASmL1dhB0ng7kyUG7CHylD9CsoYnvKPhWni1K9pYlY3W4dmVXR9rJodNf+P56A7zVWnzRYjrtLXIsLhAwx0adFIM6cXsLC4tNUhp7Ho3dHejcTtcYLDYl3rI5VikDQMk2ZzNJqwiXWSxcVksJTVYis4xeYX1l3v/H9bDgnApvhhUhQSwDby2iXVC3AoBaeDQ5jyxVkJAJLexg4IAlgxID6YTuOPZYKagVu+A3Z3fvi10a4QFESrdWYIEdRwVI2J5u88kRlnjtuO/p56emgwe/jNoMtyDdZy7VqFAoY015ATrne0O8AbmALKSm6Dc+HXW5g2Z5A2B7hCqnJC6RIJp2drDLq1Hz1Mzf64SCb2ZKAB8mCjt9EesvxWvnL5NtWg4Lt09vYNtQIb ...
```
FYI
`config/master.key` に対応する秘匿情報の設定ファイルです。
当然`key`ファイルがアップロードされるとまずいので、デフォルトでは`.gitignore`に初めから追加されます。

shmn7iii replied:
ほんとですね見落としてました...
Heroku連携の関係でファイル構造をいじり倒した際に、railsのデフォルト`.gitignore`とは別の`.gitignore`と取り違えてしまったことが原因だと思います。
ありがとうございます！

p0pemaru reviewed (commented):
ディレクトリ構造もgithubに上げるために`.keep`ファイルを追加してくれたかと思いますが、今回は無くとも問題無いです！

shmn7iii committed:
add final lines

これ便利：https://qiita.com/norikt/items/83674fadd79a88bf7824

ちなみに末尾に空行を入れる理由は：https://qiita.com/Aseiide/items/e66be44c4b7972902063

shmn7iii committed:
update gitignore to rails format

すでにコミットしちゃったファイル群は諦めます😔

p0pemaru reviewed (approved):
LGTM!
